docs/releasing.md

```diff
@@ -1,116 +1,158 @@
+git push origin capm3-v1.x.y
+
+# Mariadb
+git clone git@github.com:metal3-io/mariadb-image.git
+# repeat the steps above
+```
+
+After this, [Ironic](https://quay.io/repository/metal3-io/ironic) and
+[Mariadb](https://quay.io/repository/metal3-io/mariadb) container images will be
+built automatically with `capm3-v1.x.y` tag in Quay.
+
+### Release artifacts
+
+We need to verify all release artifacts are correctly built or generated by
+the release workflow. For a release, we should have the following artefacts:
+
+Git tags pushed:
 
-   ```bash
-   git push origin v0.x.y
-   git push origin test/v0.x.y
-   ```
+- Primary release tag: `v1.x.y`
+- Go module tags: `api/v1.x.y` and `test/v1.x.y`
 
-   NB: `origin` should be the name of the remote pointing to
-   `github.com/metal3-io/cluster-api-provider-metal3`
-1. Run `make release` to build artifacts (the image is automatically built by CI)
-1. [Create a release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
-   that contains the elements listed above that have been created in the `out`
-   folder
-1. Create a branch `release-0.x` for a minor release for backports and bug fixes.
-   This is not needed for patch releases.
+Container images built and tagged at Quay registry:
 
-### Permissions
+- [cluster-api-provider-metal3:v1.x.y](https://quay.io/repository/metal3-io/cluster-api-provider-metal3?tab=tags)
+- [ironic:capm3-v1.x.y](https://quay.io/repository/metal3-io/ironic?tab=tags)
+- [mariadb:capm3-v1.x.y](https://quay.io/repository/metal3-io/mariadb?tab=tags)
 
-Releasing requires a particular set of permissions.
+TODO: finish discussion if/how to integrate non-CAPM3 parts here.
 
-- Tag push access to the GitHub repository
-- GitHub Release creation access
+Files included in the release page:
 
-## Impact on Metal3
+- A manifest file - `infrastructure-components.yaml`
+- A metadata file - `metadata.yaml`
+- A cluster template - `cluster-template.yaml`
+- A file containing an example of variables to set - `example_variables.rc`
 
-Multiple additional actions are required in the Metal3 project
+### Release notes
+
+Next step is to clean up the release note manually.
+
+- Check for duplicates, reverts, and incorrect classifications of PRs, and
+  whatever release creation tagged to be manually checked.
+- For any superseded PRs (like CAPI uplifts, or commit revertions) that provide
+  no value to the release, create a summary line in `Superseded` section with
+  the PR ids and summary title. This way the changes are acknowledged to be part
+  of the release, but not overwhelming the important changes contained by the
+  release.
+- Make sure Ironic and Mariadb version references are in place with the correct
+  version numbers.
+- If the release you're making isn't the latest release (new
+  minor or major, or new patch release in the latest minor release),
+  uncheck the box for latest release.
```

Comment:
This is still confusing to me, though we discussed it earlier iterations, and also as you pointed out, when:

`you release from top off the latest release branch, otherwise must be unticked. Release from 1.2 branch will never be latest if you have 1.3 releases.`

But above, you are mention both new minor and major which should be ticked as latest when you publish one

Reply:
Fixed, better now?

Reply:
Thanks, it looks a lot better

Reply:
Feel free to resolve it